src/ModuleList/Article/ArticleStyles.ts

```diff
@@ -0,0 +1,32 @@
+import styled from '@emotion/styled';
+
+export const MarkdownWrapper = styled('h3')`
+  color: white;
+  width: 80%;
+  line-height: 1.5rem;
+  p {
+    margin-bottom: 1rem;
+  }
+`;
+
+export const HeadingContainer = styled('h1')`
+  color: white;
+  width: 80%;
+  margin-bottom: 1rem;
+
+`;
+
+export const AuthorContainer = styled('h1')`
+  color: white;
+  width: 80%;
+  margin-bottom: 1rem;
+
+`;
```

Comment:
Both of these styled-components have the same css properties so I only think we need one of them. One thing you could actually do since you used Chakra is to edit the `Heading` component from Chakra. See below... Then instead of using `Heading` and the `*Container` styled-components, you would just use the `ArticleHeader` component with the same props that was passed in before.
```suggestion
import { Heading } from '@chackra/ui';
export const ArticleHeader = styled(Heading)`
  color: white;
  width: 80%;
  margin-bottom: 1rem;
`;
```